db/sql/rounds/summed_round.sql

```diff
@@ -16,7 +16,9 @@
 /*
   DESCRIPTION: ?
 
-  PARAMETERS: ?
+  PARAMETERS:
+  	$1 - activeDelegates - Number of active delegates
+  	$2 - round - Round number
 */
 
 SELECT sum(r.fee)::bigint AS fees, array_agg(r.reward) AS rewards, array_agg(r.pk) AS delegates
```

Comment:
Since we are changing the SQL, could we, please move into Named Parameters at the same time?